html/includes/api_functions.inc.php

```diff
@@ -14,10 +14,12 @@
 
 function authToken(\Slim\Route $route)
 {
+    global $config;
+
     $app   = \Slim\Slim::getInstance();
     $token = $app->request->headers->get('X-Auth-Token');
     if (isset($token) && !empty($token)) {
-        if (!function_exists('get_user')) {
+        if (!function_exists('get_user') || $config['api_auth_local'] === true) {
```

Comment:
Can you switch to using `Config::get()` for this:

https://github.com/murrant/librenms/blob/5f1a1fc1e7da177374cce16802706ccfcba03e36/html/includes/authentication/active_directory.inc.php#L39